site/test/ColorCalculator.php

```diff
@@ -0,0 +1,79 @@
+<?php
+include('config/init.php');
+
+
+echo "
+	<form action='' method='get'>
+		<select name='Color1'>
+			<option value='Red'>
```

Comment:
Minor formatting thing: If you're going to put the text inside the <option> tag on a new line, you should probably also indent it. Right now it's a bit hard to scan all these options.